jenkins/databricks/clusterutils.py

```diff
@@ -85,6 +85,8 @@ def wait_for_cluster_start(workspace, clusterid, token, retries=20, printLoc=sys
             if current_state in ['INTERNAL_ERROR', 'SKIPPED', 'TERMINATED'] or p >= 60:
                 if p >= retries:
                    print("Waited %d times already, stopping" % p)
+                # Output the cluster ID to stdout so a calling script can get it easily
+                print(clusterid, file=sys.stdout)
```

Comment:
Let us add the context of what is being logged 

```suggestion
                print(f"{clusterid=}", file=sys.stdout)
```


Explicitly specifying sys.stdout is inconsistent with the print statement just preceding this one. Should we update all sys.stdout prints for consistency?